### PR TITLE
[bitnami/mysql] Create shared service for mysql

### PR DESCRIPTION
**Pull request by nobiit**
### Description of the change
Create a service that will include both primary and secondary

### Benefits
Separating primary and secondary is great if implemented. But sometimes they are not useful. For example, in an old application, only 1 URI could be specified for the database. Creating a service that includes both Primary and Secondary will be useful for that case

### Possible drawbacks
None

### Applicable issues
None

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by javsalgar** [comment]
Thanks for the PR! Could you fix the merge issue?

**Comment by nobiit** [comment]
Hey @javsalgar , I fixed the conflict. Can you review it again ?

**Comment by nobiit** [comment]
Hey @javsalgar .Can you review it again ?

File: bitnami/mysql/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: mysql
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mysql
   - https://mysql.com
-version: 9.2.5
+version: 9.2.6
```
Comment (javsalgar) [inline bitnami/mysql/Chart.yaml:28]: As it is adding a non-breaking feature, could you bump the minor version?

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thank you! Minor change

**Pull request closed without merging**